template/install.rdf

```diff
@@ -18,7 +18,12 @@
 
     <!-- Front End MetaData -->
     <em:name>Valence</em:name>
-    <em:description>Protocol Adapters for Firefox Developer Tools</em:description>
+    <em:description>Protocol Adapters for Firefox Developer Tools. User documentation is available on MDN: https://developer.mozilla.org/docs/Tools/Valence
```

Comment:
Once upon a time, I thought I had seen a way to use limited HTML tags here... There is a `fullDescription` field, but it seems to only pull from AMO. Oh well.